www/Media.js

```diff
@@ -170,11 +170,7 @@ Media.prototype.setVolume = function(volume) {
  * Adjust the playback rate.
  */
 Media.prototype.setRate = function(rate) {
-    if (cordova.platformId === 'ios'){
-        exec(null, null, "Media", "setRate", [this.id, rate]);
-    } else {
-        console.warn('media.setRate method is currently not supported for', cordova.platformId, 'platform.');
-    }
+    exec(null, null, "Media", "setRate", [this.id, rate]);
```

Comment:
This applies to all platforms now (incl. WP, still), however the actual support is only for iOS and Android, as far as I understand.